README.md

```diff
@@ -0,0 +1,44 @@
+[![CRAN_Status_Badge](http://www.r-pkg.org/badges/version/text2vec)](http://cran.r-project.org/package=text2vec)
+[![Travis-CI Build Status](https://travis-ci.org/dselivanov/text2vec.svg?branch=master)](https://travis-ci.org/dselivanov/text2vec)
+[![License](http://img.shields.io/:license-mit-blue.svg?style=flat)](http://badges.mit-license.org)
+[![Downloads](http://cranlogs.r-pkg.org/badges/grand-total/text2vec)](http://cran.rstudio.com/package=text2vec)
+[![Follow](https://img.shields.io/twitter/follow/dselvan0v.svg?style=social)](https://twitter.com/intent/follow?screen_name=dselvan0v)
+
+# Tutorials
+
+1. [Vectorization](http://dsnotes.com/blog/text2vec/2015/11/09/text2vec/).
+2. [GloVe on English Wikipedia](http://dsnotes.com/blog/text2vec/2015/12/01/glove-enwiki/).
+
+# Features
+
+**text2vec** is a package that provides an efficient framework with a concise API for text analysis and natural language processing (NLP) in R. It is inspired by [gensim](http://radimrehurek.com/gensim/), an excellent Python library for NLP.
+
+The core functionality at the moment includes
+
+1. Fast text vectorization on arbitrary n-grams, using vocabulary or feature hashing.
+2. State-of-the-art [GloVe](http://www-nlp.stanford.edu/projects/glove/) word embeddings.
+
+## Efficiency  
+
+The core of this package is carefully written in C++, which means text2vec is fast and memory friendly. Some parts (GloVe training) are fully parallelized using the excellent [RcppParallel](http://rcppcore.github.io/RcppParallel/) package. This means that parallel processing works on OS X, Linux, Windows and Solaris (x86) without any additional hacking or tricks. In addition, there is a higher-level parallelization for text vectorization and vocabulary construction on top of the [foreach](https://cran.r-project.org/package=foreach) package, and text2vec has a streaming API so that users don't have to load all of the data into RAM.
+
+## API
+
+The API is built around the [iterator](https://en.wikipedia.org/wiki/Iterator) abstraction. The API is concise, providing only a few functions which do their job well. The package does not (and probably will not in the future) provide trivial very high-level functions. But other packages can build on top of the framework that text2vec provides.
+
+## Terminology 
```

Comment:
can delete whole section,  I copied all the gory technical details to [wiki](https://github.com/dselivanov/text2vec/wiki)